packages/server/src/course/ical.service.ts

```diff
@@ -155,9 +157,13 @@ export class IcalService {
       }).save();
     }
 
+    const officeHoursEventRegex = /\b^(OH|Hours)\b/;
```

Comment:
Is there a reason to make this regex the same as the `testRegex` from above?

Reply:
Oh we definitely don't need this now that `testRegex` has a default eh